torchvision/prototype/transforms/_augment.py

```diff
@@ -223,7 +223,7 @@ def _copy_paste(
         # This is something different to TF implementation we introduced here as
         # originally the algorithm works on equal-sized data
         # (for example, coming from LSJ data augmentations)
-        size1 = image.shape[-2:]
+        size1 = cast(List[int], image.shape[-2:])
```

Comment:
mypy started to randomly complain once the image type was defined, so I choose the cast it. Happy to ignore for performance reasons.